arkouda/dtypes.py

```diff
@@ -19,22 +19,23 @@
                          'np.float64': 'f'}
 
 dtype = np.dtype
-bool = np.dtype(np.bool)
+bool = np.dtype(bool)
 int64 = np.dtype(np.int64)
 float64 = np.dtype(np.float64)
 uint8 = np.dtype(np.uint8)
 str_ = np.dtype(np.str_)
-npstr = np.dtype(np.str)
+npstr = np.dtype(str)
 
 # Union aliases used for static and runtime type checking
-bool_scalars = Union[builtins.bool,np.bool]
+bool_scalars = Union[builtins.bool, np.bool_]
 float_scalars = Union[float,np.float64]
 int_scalars = Union[int,np.int64]
-numeric_scalars = Union[float,np.float64,int,np.int64]
-numpy_scalars = Union[np.float64,np.int64,np.bool,np.uint8,np.str,np.str_]
-str_scalars = Union[str, np.str, np.str_]
+numeric_scalars = Union[float,np.float64,int,np.int64,np.uint8]
```

Comment:
I just spoke with @glitch about this. This PR adds `np.uint8`  to `numeric_scalars` which seems right to me. But I wanted to double check with someone who's been around longer to make sure it's omission was an oversight and it wasn't intentionally excluded. Maybe @reuster986 or @hokiegeek2 can verify

Reply:
@pierce314159 good question! IIRC, just an oversight

Reply:
yeah, I think it was just an oversight. Ideally, we would accept all numpy dtypes as scalar input (except for maybe uint64), because they're all castable to something arkouda supports, but that's a discussion for another PR :)